techniques/systemSettings/remoteAccess/sshKeyDistribution/3.0/sshKeyDistribution.st

```diff
@@ -52,9 +52,13 @@ bundle agent check_ssh_key_distribution
       "homedir[${sshkey_distribution_index}]"
         string  => "${userarray_${sshkey_distribution_index}[${sshkey_distribution_name[${sshkey_distribution_index}]}][5]}";
 
+      "ssh_types" string => "ssh-rsa|ssh-dss|ssh-ed25519|ssh-ecdsa";
```

Comment:
Could you add the list of supported key types to the parameter doc in metadata.xml?

Reply:
I also add ssh-dsa which may be still in use even if it is deprecated.